cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -97,8 +97,13 @@ public void create(String appName, ApplicationBean bean, boolean shouldStart) th
         }
 
         try {
+            Log.debug("Create app:", appName,
+                    "\n bean of the app:", bean,
+                    "\n should start:", shouldStart);
+
             doCreate(appName, bean, shouldStart);
         } catch (RuntimeException e) {
+            Log.debug("Clean up the app you tried to create.");
```

Comment:
No need to terminate log messages in any kind of punctuation.

Reply:
Removed the dot